CONTRIBUTING.md

```diff
@@ -1,9 +1,15 @@
 # Contributing to FunBot
 
+## Issue Filing Convention
+ 1. It is always welcomed to post an issue.
+ 2. To get proceed, at least 3 approvals(👍) from the members are required.
+
 ## Pull Request Merging Convention
  1. All the comments SHOULD to be addressed prior to merging.
- 2. At least 3 reviewers SHOULD approve 
-  - To express an approval, press 👍  on the decription.
+  - To express an objection, the reviewer SHOULD emoji 👎 on the description.
```

Comment:
Yup, I will stress that point. 
And.. I have used emoji as a verb. It's not a formal way of writing though. :)